Samples/TrendingMovies/TrendingMovies/SentryBenchmarking.mm

```diff
@@ -0,0 +1,242 @@
+#import "SentryBenchmarking.h"
+#include <chrono>
+#include <mach/clock.h>
+#include <mach/mach.h>
+#include <pthread.h>
+#include <string>
+#import <thread>
+
+//#define SENTRY_BENCHMARK_GCD_BASED
+//#define SENTRY_BENCHMARK_NSTHREAD_BASED
+#define SENTRY_BENCHMARK_PTHREAD_BASED
+
+#define SENTRY_BENCHMARKING_THREAD_NAME "io.sentry.BenchmarkingSamplingProfiler"
+
+namespace {
+/// @note: Implementation ported from @c SentryThreadHandle.hpp .
```

Comment:
If this is ported from SentryThreadHandle.hpp I guess this was already reviewed, so I'm not going to review it again.